hack/run-e2e-tests-kind.sh

```diff
@@ -35,8 +33,8 @@ done
 
 make ginkgo
 
+# NOTE: We must ensure the outcome of the following command dictates the exit code for this script.
+#       We achieve this by ensuring it is the last command we run, so the exit code of that command
+#       will become the exit code of this script.
 # We skip the helpers package as those tests assumes the environment variable USE_CERT_MANAGER is not set.
 USE_CERTMANAGER=true TEST_USE_EXISTING_CLUSTER=true $ginkgo --always-emit-ginkgo-writer -slow-spec-threshold=5s -timeout 90m -nodes=$ginkgo_nodes --skip-package helpers -race -v ./... -covermode=count -coverprofile cover.out -progress | tee /proc/1/fd/1
-
-end=$(date +%s)
-echo "Running e2e tests took $((end-start)) seconds"
```

Comment:
Couldn't we just capture the exit-code with `GINKO_EXIT_CODE=$?` and then just end the script with `exit $GINKO_EXIT_CODE`.

Reply:
Good point, let's do that.